wireapp/wire-android
# feat: Handle guest link feature flag

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Description

This PR aims to handle guest link feature flag by disabling/enabling it if it's changed from admin console.

A dialog will be displayed if a config change is detected 


### Dependencies (Optional)

Needs to be merged after [feat: add feature flag for guest room link (AR-2452)](https://github.com/wireapp/kalium/pull/1524)

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

When you disable/enable this feature from admin console you should see a popup informing you about this change.
And Create guest link button will be disabled

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 



----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
